spec/integration/kibana/spec.yml

```
@@ -135,7 +144,7 @@ spec:
       contentMediaType: "application/json"
       pattern: '^{PACKAGE_NAME}-.+\.json$'
       forbiddenPatterns:
-        - '^.+-(ecs|ECS)\.json$' # ECS suffix is forbidden 
+        - '^.+-(ecs|ECS)\.json$' # ECS suffix is forbidden
 versions:
```

We should add a version patch so this cannot be used in older versions of the spec.
```suggestion
versions:
  - before: 3.4.0
    patch:
      - op: remove
        path: "/contents/6" # remove AI prompt definitions
```

---

Great, thanks for the ping.

So we can close this PR and continue in https://github.com/elastic/package-spec/pull/871? :)

---

@jsoriano I'm still confused about the `6`. Since I'm adding it after SLO, wouldn't it be `14`?